scripts/new-upstream-snapshot

```diff
@@ -355,11 +302,11 @@ main() {
             done
             git commit -m "$msg" "$dpseries" $drops
 
-            msg="  * drop the following cherry-picks now included:"
+            msg="drop the following cherry-picks now included:"
             for file in $drops; do
-                msg="${msg}$CR    + ${file##*/}"
+                msg="${msg}$CR+${file##*/}"
             done
-            add_changelog_entries "$msg" ||
+            python3 $(dirname $0)/add_changelog.py "${msg}" "${orig_hash}" "${bug_refs_in_clog}" ||
```

Comment:
```suggestion
            python3 "$(dirname "$0")"/add_changelog.py "${msg}" "${orig_hash}" "${bug_refs_in_clog}" ||
```
To prevent word splitting and globbing.

And I think the next command (fail) should be after `||`, otherwise:

If an error occurs in the python command, then this line is going to wait for user input and fail if the user-introduced command is faulty without informing about what failed. Or I am missing something?

Reply:
+1 @aciba90 on this I just ran shellcheck scripts/new-upstream-snapshot too and resolved most errors and warnings.
Per your second comment The || is followed by a `fail "failed adding changelog entry cherry-pick drops.` on the next line which gets called in the event of failure. So I think that part is still good.